src/GitVersion.Configuration/SupportedWorkflows/GitFlow/v1.yml

```
@@ -10,6 +10,7 @@ commit-date-format: yyyy-MM-dd
 merge-message-formats: {}
 update-build-number: true
 semantic-version-format: Strict
+versioning-strategy: 'ConfigNext,MergeMessage,TaggedCommit,TrackReleaseBranches,VersionInBranchName'
```

YAML supports arrays, so we should use those instead.

```suggestion
versioning-strategy: [ConfigNext, MergeMessage, TaggedCommit, TrackReleaseBranches, VersionInBranchName]
```

---

But this is not an array it is a flags enum. Should I change the type from `VersionStrategies` to `VersionStrategies[]`? It feels a little bit repeated unnecessarily.

---

done